### PR TITLE
Update unprocessed email docs

### PR DESCRIPTION
**Pull request by 1pretz1**
Adds new latency thresholds and updates the doc with some useful rake
tasks that were recently added.

Previews:

* https://github.com/alphagov/govuk-developer-docs/blob/update-unprocessed-alerts-docs/source/manual/alerts/email-alert-api-unprocessed-messages.html.md
* https://github.com/alphagov/govuk-developer-docs/blob/update-unprocessed-alerts-docs/source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md

---

Depends on:
https://github.com/alphagov/email-alert-api/compare/increase-latency-thresholds?expand=1

Trello:
https://trello.com/c/Li8Zo03f/381-relax-the-thresholds-for-content-change-and-message-alerts

### CODE REVIEW

File: source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md [hunk 1]
```diff
@@ -63,6 +63,14 @@ ContentChange.where("created_at < ?", 10.minutes.ago).where(processed_at: nil).m
 
 - [content_change_failed_emails][]
 
+#### Resend failed emails by email ids
+
+- [resend_failed_emails:by_id][]
+
+#### Resend failed emails by date range
+
+- [resend_failed_emails:by_date['from, to']][]
```
Comment (ChrisBAshton) [inline source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md:72]: This markdown has come out oddly.

Comment (1pretz1) [reply source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md]: The links should be rendering correctly now. Good spot 👁️

File: source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md [hunk 2]
```diff
@@ -18,8 +18,13 @@ This may be fine and the emails will eventually go out, but it's worth some inve
...
 * `critical` - `content_changes` unprocessed for over 120 minutes
 
-See the [ProcessContentChangeAndGenerateEmailsWorker][content-change-worker] for
-more information.
+Every 30 minutes, [RecoverLostJobsWorker][recover-lost-jobs-worker] automatically requeues jobs for any
+unprocessed content changes and messages that are over 1-hour old.
+
+To initiate recovery immediately, run this rake task manually: [recovery:recover_lost_jobs][].
+
+See the [ProcessContentChangeWorker][content-change-worker]
```
Comment (benthorner) [inline source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md:26]: Dropping people straight into the worker is quite "heavy".

How about linking to [here](https://docs.publishing.service.gov.uk/manual/email-notifications-how-they-work.html) and (also) expanding those docs with details about these workers?

File: source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md [hunk 3]
```diff
@@ -71,13 +70,20 @@ ContentChange.where("created_at < ?", 120.minutes.ago).where(processed_at: nil).
 
 - [resend_failed_emails:by_date][]
 
+#### Requeue all unprocessed content changes and messages that are over 1-hour old
+
+- [recovery:recover_lost_jobs][]
```
Comment (benthorner) [inline source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md:75]: We shouldn't need this here, as it's not an informational task.

File: source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md (hunk 2 above)
Comment (laurentqro) [reply source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md:26]: Good idea. I could also add a section in here: https://docs.publishing.service.gov.uk/apis/email-alert-api/troubleshooting.html.

Comment (laurentqro) [reply source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md:26]: https://github.com/alphagov/email-alert-api/pull/1318/files#diff-2ff67fee0173608876ccc012bf3ff6ceR81

Comment (laurentqro) [reply source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md]: https://docs.publishing.service.gov.uk/manual/email-notifications-how-they-work.html is high-level presentation information about the app, so I thought I'd link directly to https://docs.publishing.service.gov.uk/apis/email-alert-api/troubleshooting.html.

File: source/manual/alerts/email-alert-api-unprocessed-messages.html.md [hunk 4]
```diff
@@ -20,7 +20,12 @@ would expect. This may be fine and the emails will eventually go out, but it's w
 * `warning` - unprocessed `messages` created more than 90 minutes ago
 * `critical` - unprocessed `messages` created more than 120 minutes ago
 
-See the [ProcessMessageWorker][process-message-worker] for more information.
+Every 30 minutes, [RecoverLostJobsWorker][recover-lost-jobs-worker] automatically requeues jobs for any
+unprocessed content changes and messages that are over 1-hour old.
+
+To initiate recovery immediately, run this rake task manually: [recovery:recover_lost_jobs][].
+
+For more information, see [Email Alert API troubleshooting][].
```
Comment (laurentqro) [inline source/manual/alerts/email-alert-api-unprocessed-messages.html.md:28]: => https://github.com/alphagov/email-alert-api/pull/1318/commits/37ea9502578d514bc58d11507aba89c9171087f9

File: source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md [hunk 5]
```diff
@@ -63,11 +64,23 @@ ContentChange.where("created_at < ?", 10.minutes.ago).where(processed_at: nil).m
...
 - [content_change_failed_emails][]
 
+#### Resend failed emails by email ids
+
+- [resend_failed_emails:by_id][]
+
+#### Resend failed emails by date range
+
+- [resend_failed_emails:by_date][]
+
```
Comment (kevindew) [inline source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md:74]: Thanks for putting this together @laurentqro, after chatting with Ben earlier about whether rake task is a good idea or not he raised that these docs are at risk of becoming a bit of a task buffet with various code snippets that slip out of date and I was inclined to agree.

In light of this I wonder if we can cut the information down further so there is less to maintain and more focused. I've had a quick bash at reworking your words in this way, let me know if you think it'd be a good approach.

```
This alert is triggered when content changes are not processed within the time we would expect.

* `warning` - `content_changes` unprocessed for over 90 minutes
* `critical` - `content_changes` unprocessed for over 120 minutes

Every 30 minutes, [RecoverLostJobsWorker][recover-lost-jobs-worker] automatically requeues jobs for any
unprocessed content changes and messages that are over 1-hour old.

If you see this alert it is likely that something has broken in Email Alert API that is either blocking one or all recovery attempts.

To diagnose this problem you should consult [Sentry](https://sentry.io/organizations/govuk/issues/?project=202220) to see if there are errors reported. If you find nothing conclusive in Sentry you should check [Email Alert API sidekiq logs](https://docs.publishing.service.gov.uk/manual/logging.html#kibana) to check the jobs are running correctly.

If these don't yield any results you can try running the [RecoverLostJobsWorker][recover-lost-jobs-worker] and/or [ProcessContentChangeWorker] [directly](https://stackoverflow.com/a/48543738) to see if any problems occur.
```

and then we could delete both the useful queries and useful rake tasks sections (as I don't think many of them are very useful in this scenario).

Comment (laurentqro) [reply source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md:74]: Yeah completely agree @kevindew . This page is very noisy; a more focused set of instructions will be far more helpful. I'll update the wording and delete the so-called useful queries and rake tasks.

**Pull request merged**